hal/ia32/arch/cpu.h

```diff
@@ -204,6 +256,17 @@ typedef struct {
 
 #pragma pack(pop)
 
+typedef struct {
+	tss_t tss[MAX_CPU_COUNT];
+	char stacks[MAX_CPU_COUNT][SIZE_KSTACK];
+	u32 dr5;
+	unsigned int ncpus;
+	volatile unsigned int readyCount;
+	unsigned int cpus[MAX_CPU_COUNT];
+} cpu_t;
```

Comment:
This is specific for IA32 arch, so `cpu_t` identifier is way too generic and can cause name collision. Also, `hal_` prefix should be used